### ISSUE TITLE
Rename "Parachains" to "Rollups"

### ISSUE THREAD

**Issue by filippoweb3** [w3f/polkadot-wiki#6267]
Open to discussion:

- Include a page where we explain Polkadot rollups and compare it with zk and optimistic rollups
- This is part of a wider reorg about Comparisons

**Comment by CrackTheCode016** [comment]
Would this be WIk-wide, or just for comparisons? We would also have to update the terminology in `polkadot-docs`, but ultimately this kind of seems like a branding decision.

**Comment by filippoweb3** [comment]
Yeah, the change would affect the whole Wiki docs. Renaming parachins to rollups would be more generic, more in line with JAM proposition, and help Wiki discoverability.

**Comment by Overkillus** [comment]
Gav recently called parachains cynical rollups during one of the recent keynotes. I agree there is some merit to it and I also like reminding people that our parachains are kinda like rollups, close to optimistic rollups of ethereum but not as naively optimistic, hence cynical rollups.

Altohough having said that I'm not sure if this is enough to warrant a rebrand.

If we'd rename it definitely distinguish ourselves and rename to cynical rollups and not just rollups. This opens the door to present cynical rollups as a straight up upgrade over optimistic rollups which have a lot of unwritten rules and incentives. I don't think we exploit this angle enough.

Issue is parachains is something that is really well known about Polkadot and this term distinguishes us quite a lot amongst the competition, if we'd surrender the term we'd have to rebuild the brand awareness around cynical rollups. This does not come cheap. While I belive cynical rollups would maybe be a slightly better term (explains a bit more than just parachain) is it enough of a reason? 

You could maybe argue that this swap could be done during the JAM transition, rebrands at that level could be much easier to pull off. I absolutely do not suggest doing it before that.

**Comment by Overkillus** [comment]
If it is not a matter of a rename but just an alternative terminology then I think this is much more easily doable. Parachain AKA parallel parachain is a pretty unbiased term, it does not say how the chain operates at all just that this is paralell to some other chain. You could technically say that Optimism and Abitrum are "parachains" but this would not paint the whole picture. The way we implemented our parachains is thorugh a mechanism that can be described as cynical rollups.

**Comment by filippoweb3** [comment]
yes but parachains are chains, while cores will be used for stuff that are not necessarily chains. Rollup is a more generic term, that does not imply anything on what runs on a core. The term "cynical" is, in my view, bad marketing as the word is negative. The Wiki, and Polkadot in general, will gain more SEO discoverability by just using rollups, Polkadot rollups.

**Comment by kianenigma** [comment]
> Include a page where we explain Polkadot rollups and compare it with zk and optimistic rollups


https://blog.kianenigma.com/blockchain-reimagined/presentation-tum/ could act as a good resource for this.

> polkadot-docs, but ultimately this kind of seems like a branding decision.

I agree, if this change happens by any chance, it should be coordinated with at least a handful of resources: 

1. polkadot-docs
2. pba
3. polkadot-sdk-docs


---

While I notionally agree with the proposal here:

1. Rollup is a more suiting word than a Parachain
2. It is more JAM-compatible

I am a bit worried that a fast move without enough coordination would break more things than it solves.

If you can open a forum post, get very good approval from the entire ecosystem, then I would be more comfortable with this. 

If not, how about a mixture of the following: 

1. Page outlining parachains vs. Rollups
2. A "Glossary" section where certain synonym keywords, incl. "Parachain ~ Rollup" is covered. Then:
  * An extension to the website that reads the glossary, and optionally (imagine a toggle in the navbar) annotates any reference of a word in a glossary in a page with a tooltip with its definition. 

I am actually excited about the 3rd option now:

1. It is extensible, can solve other similar issues with synonym words plaguing the wiki (Node vs. Client, Substrate vs. Polkadot-SDK)
2. The glossary can act as a single source of truth. 
3. It does not radically impact the existing website
4. But since the tooltip content is in the HTML, it will hopefully nonetheless improve our SEO with terms like "Rollup".

**Comment by Polkadot-Forum** [comment]
This issue has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/renaming-parachains-to-a-more-generic-term/10467/1
